Jenkinsfile

```diff
@@ -10,7 +10,7 @@ node ('!(ci-agent-4)') {
     stage('Checkout') {
       govuk.checkoutFromGitHubWithSSH(REPOSITORY)
       govuk.cleanupGit()
-      govuk.mergeMasterBranch()
+      // govuk.mergeMasterBranch()
```

Comment:
Will this line be commented out forever, or can it be removed?

Reply:
Not sure what it does? Do we still need it? As doesn't github check for merge conflicts and suggests pulling things from master automatically? If we don't need to do this I'd be inclined to remove it instead